### ISSUE TITLE
False positives inside multiline string

### ISSUE THREAD

**Issue by SebDieBln** [jsfehler/flake8-multiline-containers#127]
I evaluated `flake8-multiline-containers 0.0.19` and running it on the following code raises two warnings:
```
foo = """
(lorem
 ) lorem ipsum!
"""
```

```
test.py:2:6: JS101 Multi-line container not broken after opening character
test.py:3:2: JS102 Multi-line container does not close on same column as opening
```

As there are no containers and hence no multi-line containers in this code I see these as false positives.
